provider/gh-provider.go

```diff
@@ -0,0 +1,58 @@
+package provider
+
+import (
+	"context"
+
+	"github.com/src-d/gitcollector"
+	"github.com/src-d/gitcollector/discovery"
+	"github.com/src-d/gitcollector/library"
+
+	"github.com/google/go-github/github"
+)
+
+// NewGHOrgProvider builds a new gitcollector.Provider
+// based on a discovery.GHDiscovery.
+func NewGHOrgProvider(
```

Comment:
Same comment as with `discovery.GHDiscovery`. This is already in `provider` package so it could be `provider.NewGitHubOrg`.